### PR TITLE
Exam management service name renamed.

### PR DESCRIPTION
**Pull request by cseshohag**

### CODE REVIEW

File: eSchool.sln [hunk 1]
```diff
@@ -27,19 +27,23 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebStatus", "src\Web\WebSta
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Libraries", "Libraries", "{74511F4E-FF9D-42C4-9531-A75C61270B73}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry", "src\Libraries\OpenTelemetry\OpenTelemetry.csproj", "{7B410F3B-36E0-4853-9B4E-41D0CC2865B5}"
```
Comment (NitaiBanik) [inline eSchool.sln:30]: Bhai, can you please restore the project's reference number that you didn't work?

**Review by cseshohag — COMMENTED** [review]
eSchool.sln removed

**Review by cseshohag — COMMENTED** [review]
Initial service implementation for ExamManagement

File: eSchool.sln [hunk 2]
```diff
@@ -41,6 +41,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Frontend.Blazor.Server", "s
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Frontend.Blazor.Shared", "src\Web\Frontend.Blazor\Frontend.Blazor.Shared\Frontend.Blazor.Shared.csproj", "{4EB86635-CF79-4D15-909E-C41C98B0B586}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CertificateProcessing", "CertificateProcessing", "{38032F91-9580-44DC-AF92-0F562D7126F8}"
```
Comment (ratanparai) [inline eSchool.sln:44]: Please reset the sln file modification. If you add your project to the solution file, then you also need to modify all the docker files.

File: src/Services/ExamManagement/ExamManagement.API/ExamManagement.API.csproj [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.1.4" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.2" />
```
Comment (ratanparai) [inline src/Services/ExamManagement/ExamManagement.API/ExamManagement.API.csproj:12]: You do not need to add fxcopanalyzers. Please remove the package.

File: src/Services/ExamManagement/ExamManagement.API/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+
+            }
+            app.UseSwagger();
+            app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "ExamManagement.API v1"));
+
+            app.UseHttpsRedirection();
```
Comment (ratanparai) [inline src/Services/ExamManagement/ExamManagement.API/Startup.cs:41]: Please remove the https redirection

**Review by ratanparai — CHANGES_REQUESTED** [review]

File: eSchool.sln (hunk 2 above)
Comment (cseshohag) [reply eSchool.sln]: Modified the eSchool.sln file.

File: src/Services/ExamManagement/ExamManagement.API/Startup.cs (hunk 4 above)
Comment (cseshohag) [reply src/Services/ExamManagement/ExamManagement.API/Startup.cs]: Removed https redirection from the startup file.

File: src/Services/ExamManagement/ExamManagement.API/ExamManagement.API.csproj (hunk 3 above)
Comment (cseshohag) [reply src/Services/ExamManagement/ExamManagement.API/ExamManagement.API.csproj]: Removed fxcopanalyzers package.

**Review by ratanparai — APPROVED** [review]

**Pull request merged**